grafana-plugin/src/styles/utils.styles.ts

```diff
@@ -26,6 +31,37 @@ export const getUtilStyles = (theme: GrafanaTheme2) => {
     cursorDefault: css`
       cursor: default;
     `,
+
+    overflowChild: css`
+      overflow: hidden;
+      text-overflow: ellipsis;
+      display: -webkit-box;
+      white-space: initial;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
+
+      &--line-1 {
+        -webkit-line-clamp: 1;
+      }
+
+      &--line-3 {
+        -webkit-line-clamp: 3;
+      }
+    `,
+
+    wordBreakAll: css`
+      word-break: break-all;
+    `,
+
+    flex: css`
+      display: flex !important;
```

Comment:
can we remove `!important` here? it considers a bad practive

Reply:
Hmm I believe this is needed if you want to overwrite the display. Why is it considered a bad practice in this case? The utils are not too specific (as in `div.className`) and they need to overwrite some properties

Reply:
https://stackoverflow.com/a/3706876 came across this post on stackoverflow mentioning the usage of !important within utils classes